server/core/templates/credentials_update_partial.html

```diff
@@ -112,26 +112,28 @@
             (% let primary_state = primary_state %)
             (% include "credentials_update_primary.html" %)
 
+            <hr class="my-4" />
             <div id="cred-update-commit-bar" class="toast" role="alert"
                 aria-live="assertive" aria-atomic="true">
                 <div class="toast-body">
                     <span class="d-flex align-items-center">
+                        <div>
                         <svg xmlns="http://www.w3.org/2000/svg" width="16"
                             height="16" fill="currentColor"
                             class="bi bi-floppy2-fill" viewBox="0 0 16 16">
                             <path d="M12 2h-2v3h2z" />
                             <path
                                 d="M1.5 0A1.5 1.5 0 0 0 0 1.5v13A1.5 1.5 0 0 0 1.5 16h13a1.5 1.5 0 0 0 1.5-1.5V2.914a1.5 1.5 0 0 0-.44-1.06L14.147.439A1.5 1.5 0 0 0 13.086 0zM4 6a1 1 0 0 1-1-1V1h10v4a1 1 0 0 1-1 1zM3 9h10a1 1 0 0 1 1 1v5H2v-5a1 1 0 0 1 1-1" />
                         </svg>
-                        <b class="px-1">Careful</b>- save when you're done:
+                        <b class="px-1">Careful</b>- Unsaved changes will be lost</div>
```

Comment:
Spaces are better than x-spacing 😄 
```suggestion
                        <b>Careful</b> - Unsaved changes will be lost</div>
```